packages/cli/src/commands/sandbox/sandbox_event_handler_factory.test.ts

```diff
@@ -171,15 +171,12 @@ void describe('sandbox_event_handler_factory', () => {
         .successfulDeployment.map((e) => e())
     );
 
-    assert.deepStrictEqual(
+    assert.match(
       printMock.mock.calls[0].arguments[0],
-      'Amplify configuration could not be generated.'
+      /Amplify configuration could not be generated./
     );
 
-    assert.deepStrictEqual(
-      printMock.mock.calls[1].arguments[0],
-      'test error message'
-    );
+    assert.match(printMock.mock.calls[1].arguments[0], /test error message/);
```

Comment:
These changes were because the ANSI Escape codes for the colors were showing up and causing the equality to be false